plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTablePartitioning.java

```diff
@@ -16,32 +16,72 @@
 import com.google.common.collect.ImmutableList;
 import io.trino.metastore.HiveBucketProperty;
 import io.trino.metastore.SortingColumn;
-import io.trino.plugin.hive.util.HiveBucketing.BucketingVersion;
+import io.trino.plugin.hive.util.HiveBucketing;
 
 import java.util.List;
 
 import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 import static java.util.stream.Collectors.toList;
 
 /**
+ * @param active if not set partitioning is disabled for reading
```

Comment:
comment seems a bit unclear, maybe `If not set, partitioning is disabled for reading`